File: Purge.rb
#Author : yayahc | Purge Function | Version 1.2

def purge(string, string_to_purge)
    
    w = []
    a = string.chars
    
    for i in a do
        if string_to_purge == i

        else
            w += [i]
        end

    end

    return w.join()
end


puts(purge("welcome", "o"))

File: Purge.py
#Author : yayahc | Purge Function | Version 2.1

def purge(string, string_to_purge):

    a = list(string)

    for i in a:

        if string_to_purge == i:
            pass

        else :
            w.append(i)

    return "".join(w)

w = []
print(purge('welcome','o'))